deploy/openshift4/nsx-ipi-helper.yaml

```diff
@@ -0,0 +1,49 @@
+# nsx-ipi-helper DaemonSet
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nsx-ipi-helper
+  namespace: nsx-system-operator
+  labels:
+    tier: nsx-networking
+    component: nsx-ipi-helper
+    version: v1
+spec:
+  selector:
+    matchLabels:
+      tier: nsx-networking
+      component: nsx-ipi-helper
+      version: v1
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: nsx-networking
+        component: nsx-ipi-helper
+        version: v1
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          effect: NoSchedule
+        - key: node.kubernetes.io/unreachable
+          effect: NoSchedule
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      initContainers:
+        - name: nsx-ipi-helper
+          image: registry.access.redhat.com/ubi8/ubi:latest
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/bash", "-c", "yum -y install iptables; echo Disabling the DNS in localhost; iptables -A INPUT -p udp --dport 53 -j DROP; iptables -S INPUT; while ! curl localhost:6443 ; do sleep 1 ; done; echo Enabling the DNS in localhost; iptables -D INPUT -p udp --dport 53 -j DROP; iptables -S INPUT; echo Done"]
```

Comment:
I think using `&&` instead of `;` to execute instructions will be better since we are not using a script and we do not fall into unknown erroneous state. Alternatively, you can convert this into a small script

Reply:
Adding a script means we need to maintain a Dockerfile for IPI installer support.

Reply:
`&&` looks better I'll try it.